pom.xml

```diff
@@ -145,6 +145,18 @@
         <groupId>org.slf4j</groupId>
         <artifactId>log4j-over-slf4j</artifactId>
         <version>1.6.6</version>
+	<exclusions>
+          <!-- slf4j-api is provided by storm-core. Including slf4j-api
+	    here would cause its classes to be shaded preventing itself
```

Comment:
Ah. Good point. I dont think we need the change to `storm/pom.xml`. Let me revert  `storm/pom.xml` changes and make sure it works.